includes/internal/emails/class-email-sender.php

```diff
@@ -246,6 +246,7 @@ private function get_header_styles() {
 		}
 
 		$dom = new \DOMDocument();
+		libxml_use_internal_errors( true );
 		$dom->loadHTML( $header_content );
```

Comment:
Is it a good idea to clear the errors so they don't fill the memory?

```suggestion
		libxml_use_internal_errors( true );
		$dom->loadHTML( $header_content );
		libxml_clear_errors();
```

Reply:
Yeah, you're right, I applied your suggestion here: https://github.com/Automattic/sensei/pull/6602/commits/441910dc5b23fd868a8585913c6a2eb988c013c4